src/ast/mod.rs

```diff
@@ -2831,6 +2831,18 @@ pub enum Statement {
         to: Ident,
         with: Vec<SqlOption>,
     },
+    /// ```sql
+    /// OPTIMIZE TABLE [db.]name [ON CLUSTER cluster] [PARTITION partition | PARTITION ID 'partition_id'] [FINAL] [DEDUPLICATE [BY expression]]
+    /// ```
+    ///
+    /// See ClickHouse <https://clickhouse.com/docs/en/sql-reference/statements/optimize>
+    OptimizeTable {
+        name: ObjectName,
+        on_cluster: Option<Ident>,
+        partition: Option<Partition>,
+        include_final: bool,
```

Comment:
can this be called plain `final` (thinking similar to the volatile, temporary etc flags on a CreateTable)?

Reply:
I didn't change `include_final` to `final` since it's a reserved keyword for Rust.

Reply:
Ah makes sense!